content/docs/start/data-pipelines.md

```diff
@@ -34,7 +34,7 @@ requirements.txt  train.py
 Now let's install the requirements:
 
 > We **strongly** recommend creating a
-> [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments)
+> [virtual environment](https://python.readthedocs.io/en/stable/library/venv.html)
```

Comment:
Thanks! We should probably change this link in other places as well. I think it's in a few places.

Reply:
Moved to #1556